android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -241,8 +241,7 @@
 
   </script>
   <style type="text/css">
-      body {background-color:#000000; min-width:2500px;}
-     
+      body {background-color:rgba(0, 0, 0, 1.0);}
```

Comment:
That's almost always the default setting when dealing with plain RGB values in computer graphics, for sure.

I'd imagine we could pass along the active API version to keyboard.html and use that as a criteria for activating a legacy stylesheet, since it's possible to selectively link stylesheets through JS, rather than globally doing it for all versions.